amqtt/mqtt/protocol/client_handler.py

```diff
@@ -90,8 +91,8 @@ def handle_write_timeout(self):
             if not self._ping_task:
                 self.logger.debug("Scheduling Ping")
                 self._ping_task = asyncio.ensure_future(self.mqtt_ping())
-        except BaseException as be:
-            self.logger.debug("Exception ignored in ping task: %r" % be)
+        except Exception as e:
+            self.logger.debug("Exception ignored in ping task: %r" % e)
```

Comment:
I wonder what would trigger this

Reply:
I am a bit cautious to remove something without fully understanding it as it might break something.

Reply:
Yeah, me too actually. I restored exception handling replacing `BaseException` and postponed a refactor.